helm/node-exporter-app/templates/verticalpodautoscaler.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if and (.Capabilities.APIVersions.Has "autoscaling.k8s.io/v1") (.Values.verticalPodAutoscaler.enabled) }}
```

Comment:
I see this is coming from upstream, but just to comment - I sooo dislike these `.Capabilities.APIVersions.Has "<SOME API>"` checks in Helm charts.

It's a stateful logic in the Helm chart which can just bite us in the arse, as the chart can work differently on different occasions, depending on the state of the Kubernetes cluster in the moment of chart installation (and that state can change 5 seconds later if for some reason the CRD gets installed later).

But again, not something to change in this PR.